src/NuGet.Core/NuGet.Packaging/LocalPackageArchiveDownloader.cs

```diff
@@ -144,23 +144,21 @@ public void Dispose()
 
             cancellationToken.ThrowIfCancellationRequested();
 
+            using (var source = File.OpenRead(_packageFilePath))
```

Comment:
In my restore perf testing `CopyTo(...)` is roughly twice as slow as `CopyToAsync(...)` with all other factors the same.